python/cudf/cudf/core/dataframe.py

```diff
@@ -900,20 +900,36 @@ def get_renderable_dataframe(self):
             if len(self._data.names) > ncols:
                 right_cols = len(self._data.names) - int(ncols / 2.0) - 1
                 left_cols = int(ncols / 2.0) + 1
+
+            # Pick left_cols number of columns.
             upper_left = self.head(upper_rows).iloc[:, :left_cols]
-            upper_right = self.head(upper_rows).iloc[:, right_cols:]
+            if right_cols > 0:
+                # Pick ncols - left_cols number of columns
+                # from the right side/from the end.
+                upper_right = self.head(upper_rows).iloc[
+                    :, -(int(ncols) - left_cols + 1) :
```

Comment:
Instead of doing the math in the iloc call here, could we just do the math and set `right_cols` properly above?

Reply:
@kkraus14 Done